src/Apps/Order/Routes/__tests__/Shipping.TL.jest.tsx

```diff
@@ -217,35 +217,32 @@ describe("Shipping", () => {
         )
 
         expect(mockCommitMutation).toHaveBeenCalledTimes(2)
-        expect(mockCommitMutation).toHaveBeenNthCalledWith(
-          1,
-          expect.objectContaining({
-            variables: {
-              input: {
-                id: "1234",
-                fulfillmentType: "SHIP",
-                phoneNumber: validAddress.phoneNumber,
-                shipping: {
-                  ...validAddress,
-                  phoneNumber: "",
-                },
-              },
-            },
-          })
+
+        let mutationArg = mockCommitMutation.mock.calls[0][0]
+        expect(mutationArg.mutation.default.operation.name).toEqual(
+          "SetShippingMutation"
```

Comment:
Previously we only check the mutation variables, which can be confusing sometimes. The new pattern here uses the [`mock` property](https://jestjs.io/docs/mock-functions#mock-property), so we can more explicitly check both the mutation name and the variables.